### DISCUSSION TITLE
Cursor disappearing

### DISCUSSION THREAD

**Discussion by Truhilora** [ppy/osu#19303] [Q&A]
### Type

Game behaviour

### Bug description

Hey i have ha Problem since a few days, that makes my cursor go invisible everytime i touch the tablet with my pen. 
 
I play on Osu Lazer and i use the Gaomon S620 tablet with the Osu recomendet Driver. After reinstalling both the Driver and osu lazer it has got even worse. Now my pen gliches around while still going invisible. Playing Osu like this is impossible. 

Thanks
-Truh

### Version

Lazer 2022.716.1

### Logs

[runtime.log](https://github.com/ppy/osu/files/9169473/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9169474/updater.log)
[database.log](https://github.com/ppy/osu/files/9169475/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9169476/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9169477/network.log)
[performance.log](https://github.com/ppy/osu/files/9169478/performance.log)

**Comment by frenzibyte** [comment]
Have you tried using the in-game tablet support? Also what external driver are you using, does it feature a "Windows Ink" option somewhere?

**Reply by Truhilora** [reply]
Yes i have tried it with and without the in game tablet support but the cursor still disappears.
I couldnt find a "Windows Ink" option on my tablet driver. I'm not sure what you mean by "external driver" tho.

**Reply by frenzibyte** [reply]
External driver means the driver (aka app) you use to configure your tablet. While enabling in-game tablet support, does disabling "Touch" in settings fix the issue?

**Reply by Moltanicaa** [reply]
I got the same problem. I play on a laptop, tablet XP-pen G640. This tablet has official drivers, where touching the tablet is equal to pressing the left mouse button. There is no way to turn this off at the driver level (this driver is pretty weird). It seems to me that the osu!lazer just shouldn't make the cursor invisible.

At the same time, I have a Pentablet on my desktop PC, where there is no such problem.

**Reply by Moltanicaa** [reply]
I tried turning off "touch" and that removed the disappearing cursor, but that caused the cursor to stop moving when pressed. It just sticks to one point.

**Reply by frenzibyte** [reply]
The cursor will become visible on next release, so revert any changes you made and wait for that.

**Reply by peppy** [reply]
Note that that is a setting you may need to toggle.

**Reply by Moltanicaa** [reply]
Problem solved
